package.json

```diff
@@ -56,20 +60,28 @@
     "tmp": "0.0.33"
   },
   "devDependencies": {
+    "@types/commander": "^2.12.2",
+    "@types/node": "^16.11.7",
+    "@types/semver": "^7.3.9",
+    "@types/tape": "^4.13.2",
+    "@typescript-eslint/eslint-plugin": "^5.4.0",
+    "@typescript-eslint/parser": "^5.4.0",
     "coveralls": "^3.0.0",
+    "eslint": "^7.32.0",
+    "eslint-config-standard": "^16.0.3",
+    "eslint-plugin-import": "^2.25.3",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-promise": "^5.1.1",
```

Comment:
I think that ESLint could also be introduced separately from the TypeScript conversion? That's another thing that could be a separate PR. Given that reviewing is the main bottleneck here, smaller, more focused PRs would be really helpful.

Reply:
Yes/No, the current implementation cannot / does not support linting typescript. Additional packages would be required to do this and we would use the semicolon enforcement. So this would be the way to keep linting and keep `standard/standard` as the style guide.

Reply:
I'm not sure I understand. Why would you need extra packages to lint TypeScript if this was a separate PR still on the JS version of the project?

Reply:
So the base idea was to ensure that if we did the Typescript changes that everything would still function as expected. If we do this change and not include the eslint change then the code will no longer be linted.

We can make this change (linting) prior to the Typescript change being merged. Simplifying the typescript change since the linting process can be upgraded already?

Reply:
> We can make this change (linting) prior to the Typescript change being merged.

That's exactly what I had in mind. The TypeScript change will probably require some discussion (and feedback at least from @axic, who knows much more about the project than I do) so it makes the most sense to put it on top of anything simpler that can just go in quickly.

Reply:
Makes sense, I will action this.